README.md

```diff
@@ -44,10 +44,13 @@ To be valid, a parser needs:
 ## Usables libraries
 
 For now, our parsers only use the following libraries:
+
 - [tinycolor2](https://github.com/bgrins/TinyColor)
 - [lodash](https://github.com/lodash/lodash)
+- [svgo](https://github.com/svg/svgo)
 
 If you need another library to develop your parser:
```

Comment:
Replace "If you need another library to develop your parser:" by "Need another library to develop your parser?"